spaceship/lib/spaceship/portal/portal_client.rb

```diff
@@ -586,6 +585,18 @@ def create_certificate!(type, csr, app_id = nil, mac = false)
       parse_response(r, 'certRequest')
     end
 
+    def create_certificate_apple_pay!(type, csr, merchant_id = nil, mac = false)
```

Comment:
strange method name

Reply:
@janpio Do you have any other preferred name? This seems pretty self-explanatory to me.